packages/components/_private/overlay/style/index.less

```
@@ -1,4 +1,7 @@
+@import "../../../style/mixins/reset.less";
+
 .@{overlay-prefix} {
+  .reset-component();
```

```suggestion
  .reset-component-new();
```

reset-component 被废弃掉了